dotnet/api/Infrastructure/Data.SqlServer/Migrations/GBFlattenedMigration.cs

```diff
@@ -0,0 +1,57 @@
+﻿using AndcultureCode.CSharp.Core.Constants;
+using AndcultureCode.CSharp.Core.Utilities.Configuration;
+using AndcultureCode.CSharp.Data.SqlServer.Migrations;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+
+namespace AndcultureCode.GB.Infrastructure.Data.SqlServer.Migrations
+{
+    public partial class GBFlattenedMigration : FlattenedMigration
+    {
+        public const string InitialMigrationId = "20200603183650_InitialCreate";
+
+        // This should contain all migration ID consts listed above, in the order that they should
+        // be run.
+        public static readonly List<string> _flattenedMigrations = new List<string> {
+            InitialMigrationId
+        };
+
+        #region Private
+
+        GBApiContext _context;
+
+        #endregion Private
+
+        #region Overrides
+
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+        }
+
+        public override List<string> FlattenedMigrations
+        {
+            get
+            {
+                return _flattenedMigrations;
+            }
+        }
+
+        public override DbContext Context
```

Comment:
Suggestion: Consider use of expression bodies here

```csharp
public override DbContext Context
{
    get => _context = _context ?? new GBApiContext();
}
```

Reply:
Good call, I need to use these more often.